examples/usdt_sample/scripts/latency.c

```diff
@@ -0,0 +1,74 @@
+#include <linux/blkdev.h>
```

Comment:
Do we still need this separate file if you ended up embedding it in the Python script?